### ISSUE TITLE
Meta analysis

### ISSUE THREAD

**Issue by mcfrank** [langcog/experimentology#32]
Some thoughts for the next revision:
- [x] a bit more how-to content? Eg how to extract ES
- [x] Some discussion of mini-metas within a paper?

**Comment by ColesNicholas** [comment]
@mayamathur , shall we set up a meeting to discuss further?

**Comment by mayamathur** [comment]
MCF, we have the following now regarding scraping ES: 

Because not every paper reports standardized effect sizes -- or even means and standard deviations for every group -- this process can often involve scraping information from plots, tables, and statistical tests to try to reconstruct effect sizes. For example, if the outcome variable is continuous, we could estimate Cohen's d (see Chapter \@ref(estimation)) from group means and standard deviations reported in the paper, even without having access to raw data. This book will not cover the process of conducting a systematic literature search and extracting effect sizes, but these topics are critical to understand if you plan to conduct a meta-analysis or other evidence synthesis.

**Comment by mayamathur** [comment]
MCF, regarding mini-metas within a paper, Nicholas and I both don't have strong opinions or recommendations on them. We can look into it, but first, did you have something specific to cover?

**Comment by ColesNicholas** [comment]
Chatted with Maya about adding more background on intergroup contact, but we voted against it to keep the chapter text streamlined

**Comment by mcfrank** [comment]
Sounds good - I did like the idea of at least saying "you can do a
meta-analysis on effects within an individual paper" and then referencing
some mini-meta stuff?

Mike

**Comment by mcfrank** [comment]
outline of an internal meta analysis bit (either box or long footnote):
- internal meta can be a good way to get a more precise estimate across a few studies
- BUT you could just do a "mega-analysis" and model all the data
- AND be careful because you don't have power to estimate heterogeneity or moderation
- finally, don't use it to "rescue" non-significant studies in a data-contingent way as this is also p-hacky
- in summary, quantitative pooling is good if done with care
could highlight Philips Ong example that is the case study for Design chapter.

**Comment by ColesNicholas** [comment]
@mayamathur and @mcfrank, I just pushed a small section on internal meta-analyses. I'm still a little reluctant to jump into the topic because there's so much to unpack (particularly about the potential issues with pooled data analysis strategies). However, I tried to strike a balance between depth and length with these edits.

**Comment by mcfrank** [comment]
looks great!
